src/main/java/org/jabref/logic/cleanup/RenamePdfCleanup.java

```diff
@@ -158,4 +159,29 @@ public String getTargetFileName(LinkedFile flEntry, BibEntry entry) {
     public int getUnsuccessfulRenames() {
         return unsuccessfulRenames;
     }
+
+    /**Check to see if a file already exists in the target directory.  Search is not case sensitive.
+    *
+    * @param flEntry
+    * @param entry
+    * @return First identified path that matches an existing file.  This name can be used in subsequent calls to override the existing file.
+    */
+    public Optional<Path> fileAlreadyExists(LinkedFile flEntry, BibEntry entry) {
```

Comment:
This is maybe a bit nitpicky: Think of a more intuitive name for the method that reflects the return type. `fileAlreadyExists` sounds to me like it would return a boolean. How about `findExistingFile` or something similar?